docs/docs/internals/syntax.md

```diff
@@ -263,9 +263,10 @@ BlockStat         ::=  Import
                     |  Expr1
                     |  EndMarker
 
-ForExpr           ::=  ‘for’ (‘(’ Enumerators ‘)’ | ‘{’ Enumerators ‘}’)        ForYield(enums, expr)
-                       {nl} [‘yield’] Expr
-                    |  ‘for’ Enumerators (‘do’ Expr | ‘yield’ Expr)             ForDo(enums, expr)
+ForExpr           ::=  ‘for’ ‘(’ Enumerators0 ‘)’ {nl} [‘do‘ | ‘yield’] Expr     ForYield(enums, expr) / ForDo(enums, expr)
+                    |  ‘for’ ‘{’ Enumerators0 ‘}’ {nl} [‘do‘ | ‘yield’] Expr
+                    |  ‘for’     Enumerators0     {nl} (‘do‘ | ‘yield’) Expr
```

Comment:
Same comment about nl. Also, there's a divergence between parser and syntax.md regarding the trailing `[semi]` in Enumerators0. The syntax is also not what is supported by the parser. The parser supports trailing semicolons in braces and in braceless syntax but not in parens. That is a question we should clear up. When should enumerators suppport a trailing `;`. I believe all other proposed changes to the syntax are unnecessary.

Reply:
Finally, any PR for syntax.md needs to change both versions: in internals and in reference.

Reply:
>The parser supports trailing semicolons in braces and in braceless syntax but not in parens. That is a question we should clear up

Yeah, that I have missed...

>I believe all other proposed changes to the syntax are **UN**necessary.

Do you mean that's ok there is a divergence between parser and syntax reference? 
(syntax doesn't support `do` after `for` with brackets and new line before `do`/`yield` in braceless version

>Finally, any PR for syntax.md needs to change both versions: in internals and in reference.

The PR contains the change in both `reference/syntax.md` and `internals/syntax.md` (you've commented in both files)
What did I miss?